keyboards/handwired/onekey/keymaps/adc/keymap.c

```diff
@@ -2,10 +2,6 @@
 #include "analog.h"
 #include <stdio.h>
 
-#ifndef ADC_PIN
-#    define ADC_PIN A0
```

Comment:
without this, the f4 board now fail

Reply:
Added it back in their respective config.h's... they're still complaining it's not enabled in halconf.

Reply:
Yeah its kinda a limitation, given all functionality has to work with a single configuration. Over time we can fix this with something like #8327.